app/models/alert.rb

```diff
@@ -12,13 +12,26 @@ class Alert < ActiveRecord::Base
   scope :active, -> { where(confirmed: true, unsubscribed: false) }
   scope :in_past_week, -> { where("created_at > ?", 7.days.ago) }
 
+  before_create :attach_alert_subscriber
```

Comment:
What happens if the alert can't be saved for some reason? Does it still create the subscriber? We should check this.